src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Models/SelectEmployerViewModel.cs

```diff
@@ -4,6 +4,7 @@ namespace SFA.DAS.ProviderCommitments.Web.Models
 {
     public class SelectEmployerViewModel
     {
+        public long ProviderId { get; set; }
```

Comment:
I know this isn't your code but is it normal to have a view model with nested view models?

IList<AccountProviderLegalEntityViewModel> AccountProviderLegalEntities ???